course/include/Isometry.hpp

```diff
@@ -93,43 +93,64 @@ inline std::ostream& operator<<(std::ostream& os, const Vector3& vect) {
 }
 
 class Matrix3 {
+    using mat3 = std::vector<std::vector<double>>;
```

Comment:
Better use three `Vector3` as rows (or columns). That way you can levarage the operations  you already implemented for Vector3 for Matrix3.